### ISSUE TITLE
[CI] :build-tools:integTest failing in 7.x

### ISSUE THREAD

**Issue by matriv** [elastic/elasticsearch#46918]
Logs: 
https://gradle-enterprise.elastic.co/s/vah6pnkaggv4o/console-log?task=:build-tools:integTest 
https://gradle-enterprise.elastic.co/s/mmv2sfg2xlnow/console-log?task=:build-tools:integTest

Could not reproduce locally with:
```
./gradlew :build-tools:integTest
```

Stacktrace:
```
:build-tools:integTest FAILED
--
 
org.elasticsearch.gradle.tar.SymbolicLinkPreservingTarIT > testBZip2Tar FAILED
java.lang.AssertionError at SymbolicLinkPreservingTarIT.java:102
 
org.elasticsearch.gradle.tar.SymbolicLinkPreservingTarIT > testTarDoNotPreserveFileTimestamps FAILED
java.lang.AssertionError at SymbolicLinkPreservingTarIT.java:102
 
org.elasticsearch.gradle.tar.SymbolicLinkPreservingTarIT > testBZip2TarDoNotPreserveFileTimestamps FAILED
java.lang.AssertionError at SymbolicLinkPreservingTarIT.java:102
 
org.elasticsearch.gradle.tar.SymbolicLinkPreservingTarIT > testGZipTar FAILED
java.lang.AssertionError at SymbolicLinkPreservingTarIT.java:102
 
org.elasticsearch.gradle.tar.SymbolicLinkPreservingTarIT > testGZipTarDoNotPreserveFileTimestamps FAILED
java.lang.AssertionError at SymbolicLinkPreservingTarIT.java:102
 
org.elasticsearch.gradle.tar.SymbolicLinkPreservingTarIT > testTar FAILED
java.lang.AssertionError at SymbolicLinkPreservingTarIT.java:102
 
63 tests completed, 6 failed, 15 skipped
```

**Comment by matriv** [comment]
Also fails in 7.4: https://gradle-enterprise.elastic.co/s/3hermsru7wqia/console-log?task=:build-tools:integTest

**Comment by atorok** [comment]
This is a windows specific failure already fixed in #46908
